### PR TITLE
git-diff: add cross-branch diff example

### PR DESCRIPTION
**Pull request by sirianni**
----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      "command: add page" for new pages, or "command: description of changes" for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/git-diff.md [hunk 1]
```diff
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
+
+`git diff {{branch_1}}..{{branch_2}} [--] {{/path/to/file}}`
```
Comment (sbrl) [inline pages/common/git-diff.md]: Are the `--` optional? I thought they were required.

Also, could you remove the preceding slash in `/path/to/file` making it `path/to/file`, please?

Thanks 😺

**Comment by sbrl** [comment]
Thanks for the addition! I've found myself wanting to do something like this in the past - this would be a good addition to the page.

Please see my comment inline for suggestions for improvement 😺

File: pages/common/git-diff.md (hunk 1 above)
Comment (sirianni) [inline pages/common/git-diff.md]: >Also, could you remove the preceding slash in /path/to/file making it path/to/file, please?

Yeah sure

>Are the -- optional? I thought they were required.

They are indeed optional.  I believe they are only needed to disambiguate certain scenarios or when the file contains dashes.

https://stackoverflow.com/questions/22750028/in-git-what-does-dash-dash-mean

```
$ git --no-pager diff master..git-diff-across-branches git-diff.md           
diff --git a/pages/common/git-diff.md b/pages/common/git-diff.md
index 69a9f8c..d82e4be 100644
--- a/pages/common/git-diff.md
+++ b/pages/common/git-diff.md
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
+
+`git diff {{branch_1}}..{{branch_2}} [--] {{/path/to/file}}`
```
```
$ git --no-pager diff master..git-diff-across-branches -- git-diff.md
diff --git a/pages/common/git-diff.md b/pages/common/git-diff.md
index 69a9f8c..d82e4be 100644
--- a/pages/common/git-diff.md
+++ b/pages/common/git-diff.md
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
+
+`git diff {{branch_1}}..{{branch_2}} [--] {{/path/to/file}}`
```

File: pages/common/git-diff.md [hunk 2]
```diff
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
```
Comment (agnivade) [inline pages/common/git-diff.md]: / => or

**Comment by sbrl** [comment]
Thanks 😺

**Pull request merged**